components/commons/Navbar/Navbar.tsx

```diff
@@ -1,9 +1,24 @@
-import { FC } from "react";
-import { NavbarProps as Props } from "./Navbar.types";
-import styles from "./styles/Navbar.module.scss";
+import { FC } from 'react'
+import { NavbarProps as Props } from "./Navbar.types"
+import styles from "./styles/Navbar.module.scss"
 
 const Navbar: FC<Props> = ({}) => {
-  return <div>Navbar here</div>;
+  return (
+  <div>
+        <div className={styles.navbar}>
+            <ul>
```

Comment:
Don't use `<ul>` here. Its used when you when want to list items. Also `<ul>` can only have `<li>` as child elements. So you cannot add `img` or another `ul` inside it. Use a `div` instead.